widgets/google-map/tpl/static-map.php

```diff
@@ -14,7 +14,8 @@
 <img
 	class="sowb-google-map-static"
 	border="0"
-	src="<?php echo sow_esc_url( $src_url ) ?>"
+	src="<?php echo sow_esc_url( $src_url['desktop'] ) ?>"
+	data-src="<?php echo ! empty( $src_url['mobile'] ) ? esc_attr( json_encode( $src_url ) ) : ''; ?>"
```

Comment:
@braamgenis Should I pre-escape this in google-maps.php or should I do it here?

Reply:
The general convention we follow is to try and do escaping as late as possible, so here is good.